pyramid_jsonapi/collection_view.py

```diff
@@ -1001,7 +1001,7 @@ def object_exists(self, obj_id):
                 self.model
             ).options(
                 load_only(self.key_column.name)
-            ).get(obj_id)
+            ).filter(self.key_column == obj_id).one()
```

Comment:
According to the docs for `get()`:
  "Return an instance based on the given primary key identifier, or None if not found."
whereas `one()` will throw an exception (`NoResultFound`). Probably should either catch that in the following `except` block or `use one_or_none()` (sqlalchemy 1.0.9+ though).

Reply:
I've updated to use `one_or_none()`and simplifed the structure a little - 1.0.9 was released in 2015, and the current maintenance branch is 1.2, so there shouldn't be anyone out there relying on it!